### PR TITLE
doc: update rgw configuration doc on bucket index sharding feature

### PR DESCRIPTION
**Pull request by zhouyuan**
Signed-off-by: Yuan Zhou <yuan.zhou@intel.com>

### CODE REVIEW

File: doc/radosgw/config-ref.rst [hunk 1]
```diff
@@ -270,6 +270,17 @@ Ceph configuration file, the default value will be set automatically.
 :Default: ``1000``
 
 
+``rgw override bucket index max shards``
+
+:Description: Represents the number of shards for the bucket index object,
+              a value of zero indicates there is no sharding. It is not
+              recommended to set a too large value (e.g. thousand) as it
```
Comment (theanalyst) [inline doc/radosgw/config-ref.rst]: `set a too large value` -> `set too large a value` ?

**Comment by theanalyst** [comment]
+1

**Comment by zhouyuan** [comment]
@yehudasa could you please take a look?

**Comment by tchaikov** [comment]
@yehudasa ping ?

**Comment by tchaikov** [comment]
seems it's literally copied from the corresponding `config_opts.h` comments.

:+1:

**Pull request merged**